dv/uvm/core_ibex/sim.py

```diff
@@ -413,6 +413,43 @@ def compare(test_list, iss, output_dir):
     return fails == 0
 
 
+#TODO(udinator) - support IUS, DSim, and Riviera
+def gen_cov(base_dir, simulator, lsf_cmd):
+    """Generate a merged coverage directory, and open up the appropriate GUI
```

Comment:
As with the comment in the Makefile, can we make this two separate steps? What if I want to run it in a CI flow with no X11 connection?

Reply:
Same as above, I forgot to remove this section of the comment.